src/memberBundle.ts

```diff
@@ -107,7 +110,7 @@ export async function createMemberBundleRequest(
       publicKeyId: requestData.publicKeyId,
       memberBundleStartDate: new Date(requestData.memberBundleStartDate),
       signature: Buffer.from(requestData.signature, 'base64'),
-      awalaPda: Buffer.from(requestData.awalaPda, 'base64'),
+      peerId: requestData.peerId,
```

Comment:
I had o export CERTIFICATE_EXPIRY_DAYS to set the expiry date of the parcel. and had to rearrange the file so that the export is after the non exported function

Reply:
OK, makes sense 👍🏾